extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/config/AddressResolverConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.vertx.core.runtime.config;
+
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class AddressResolverConfiguration {
+
+    /**
+     * The maximum amount of time in seconds that a successfully resolved address will be cached.
```

Comment:
Can you describe (with words) what does the default value represent? 
It seems to be "forever"

Reply:
In the annotation?

Reply:
In the javadoc.

Reply:
I assumed so already and have pushed a corresponding change ...

Reply:
ah! Thanks!